src/Person.java

```diff
@@ -15,10 +15,43 @@ public Person(String name, LocalDate dateOfBirth, String mobileNumber, String em
     this.email = email;
   };
 
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
```

Comment:
It's generally good practice to create immutable objects: https://docs.oracle.com/javase/tutorial/essential/concurrency/immutable.html

Thus the fields should not be modifiable once the object is created, and setter should not be exposed. Have a read of the article I just posted, and try to update the `Person` object to be immutable